lms/djangoapps/philu_overrides/helpers.py

```diff
@@ -88,6 +88,10 @@ def get_course_next_classes(request, course):
         # - Course is already full
         # - Student cannot enroll in course
         active_reg_button = not (registered or is_course_full or not can_enroll)
+        target_class_link_for_staff_user = ""
+        if request.user.is_authenticated() and request.user.is_staff:
+            from openedx.core.djangoapps.timed_notification.core import get_course_first_chapter_link
+            target_class_link_for_staff_user = get_course_first_chapter_link(_course)
```

Comment:
Can we rename this to class_first_chapter_link, actually this link is going to be same for all users, beside that we are only showing it to staff.